Frameworks/UIKit/UIButton.mm

```diff
@@ -187,7 +189,7 @@ - (instancetype)initWithCoder:(NSCoder*)coder {
 
 - (void)_initUIButton {
     // Store a strongly-typed backing button
-    _xamlButton = rt_dynamic_cast<WXCButton>([self xamlElement]);
+    _xamlButton = [self _xamlElementInternal].try_as<Controls::Button>();
```

Comment:
>try_as [](start = 46, length = 6)

would .as<> throw on failure?  if so, we can probably just do that and remove the FAIL_FAST below